pmm/pmm2-upgrade-portal-tests.groovy

```diff
@@ -0,0 +1,277 @@
+        env.PMM_UI_URL = "http://${VM_IP}/"
+    }
+}
+
+void destroyStaging(IP) {
+    build job: 'aws-staging-stop', parameters: [
+        string(name: 'VM', value: IP),
+    ]
+}
+
+pipeline {
+    agent {
+        label 'docker'
+    }
+    environment {
+        PORTAL_USER_EMAIL=credentials('PORTAL_USER_EMAIL')
+        PORTAL_USER_PASSWORD=credentials('PORTAL_USER_PASSWORD')
+        OKTA_TOKEN=credentials('OKTA_TOKEN')
+        SERVICENOW_LOGIN=credentials('SERVICENOW_LOGIN')
+        SERVICENOW_PASSWORD=credentials('SERVICENOW_PASSWORD')
+        SERVICENOW_DEV_URL=credentials('SERVICENOW_DEV_URL')
+        OAUTH_DEV_CLIENT_ID=credentials('OAUTH_DEV_CLIENT_ID')
+        PORTAL_BASE_URL=credentials('PORTAL_BASE_URL')
+    }
+    parameters {
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for pmm-ui-tests repository',
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: '',
+            description: 'Commit hash for the branch',
+            name: 'GIT_COMMIT_HASH')
+        choice(
+            choices: ['2.27.0', '2.26.0', '2.25.0'],
+            description: "PMM Server Version",
+            name: 'PMM_SERVER_VERSION')
+        choice(
+            choices: ['2.27.0', '2.26.0', '2.25.0'],
+            description: "PMM Client Version",
+            name: 'PMM_CLIENT_VERSION')
+        choice(
+            choices: ['Production', 'Testing', 'Experimental'],
+            description: "Select Repository for testing",
+            name: 'PMM_REPOSITORY')
+        string(
+            defaultValue: 'admin-password',
+            description: 'Change pmm-server admin user default password.',
+            name: 'ADMIN_PASSWORD')
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for pmm-qa repository',
+            name: 'PMM_QA_GIT_BRANCH')
+    }
+    options {
+        skipDefaultCheckout()
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                // clean up workspace and fetch pmm-ui-tests repository
+                deleteDir()
+                git poll: false, branch: GIT_BRANCH, url: 'https://github.com/percona/pmm-ui-tests.git'
+                wrap([$class: 'BuildUser']) {
+                    sh """
+                        echo "\${BUILD_USER_EMAIL}" > OWNER_EMAIL
+                        echo "\${BUILD_USER_EMAIL}" | awk -F '@' '{print \$1}' > OWNER_FULL
+                    """
+                }
+                script {
+                    slackSend botUser: true, channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+                }
+                sh '''
+                    sudo yum -y update --security
+                    sudo yum -y install php php-mysqlnd php-pdo jq svn bats mysql
+                    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
We have `kubectl` in `agent-amd64` and if we need to install something it's better to add it to the agent:
https://github.com/percona/pmm-infra/blob/main/packer/ansible/agent.yml

Reply:
Removed since kubectl is in agent-amd64